pkgs/development/python-modules/license-expression/default.nix

```diff
@@ -12,6 +12,7 @@ buildPythonPackage rec {
     rev = "v${version}";
     sha256 = "0q8sha38w7ajg7ar0rmbqrwv0n58l8yzyl96cqwcbvp578fn3ir0";
   };
+  postPatch = "patchShebangs .";
```

Comment:
This could be dangerous, can we patch the exact file that needs it's shebang fixed?
If you want to see what files it patches do
```
nix-shell -A pythonPackages.license-expression
unpackPhase
cd $archive_path
patchPhase # or patchShebangs .
```